README.md

```diff
@@ -67,10 +67,43 @@ $ foreman s
 
 ### Change CMS URL Path
 
-In development, frontend will use the local CMS for convenience. You can change the MAS_CMS_URL on .env file. Use http://cms.moneyadviceservice.org.uk to point to LIVE content, https://cms.qa.dev.mas.local for testing Or http://localhost:PORT to point to a local running CMS.
+In development, frontend will use the local CMS for convenience. See [CMS repository README](https://github.com/moneyadviceservice/cms/blob/master/README.md) for instructions on setting up a local CMS instance.
+
+You can change the MAS_CMS_URL on .env file. Use http://cms.moneyadviceservice.org.uk to point to LIVE content, https://cms.qa.dev.mas.local for testing Or http://localhost:PORT to point to a local running CMS.
```

Comment:
@norm 👍  for https (I assume)

I can envisage pointing to the production CMS if there was a particularly puzzling issue that was only occurring in production. It's read-only, after all. 

That said, I have no strong opinions either way on documenting that here. Maybe worth checking with @pmcostello,  who added that suggestion, before deleting it?